src/util/report-util.js

```diff
@@ -0,0 +1,305 @@
+import flow from 'lodash/fp/flow'
+import groupBy from 'lodash/fp/groupBy'
+import entries from 'lodash/fp/entries'
+import map from 'lodash/fp/map'
+import values from 'lodash/fp/values'
+import sortBy from 'lodash/fp/sortBy'
+import pdfMake from 'pdfmake/build/pdfmake.min'
```

Comment:
Due to #21 we should try to reduce size of `pdfMake` library. It's current size is 1.6 MB. I found that can be useful - https://github.com/bpampuch/pdfmake/issues/1374. Also if possible we should reduce size of `json2csv` library too.

Currently them are too big 🐷, but if we don't find any solution to reduce the size it's ok.

Reply:
https://github.com/bpampuch/pdfmake/compare/0.1.37...master I guess we can use "pdfmake-lite" 0.1.36 since 0.1.37 didn't change that much :D